lib/mws/recommendations/client.rb

```diff
@@ -7,7 +7,8 @@ module Recommendations
     # is an actionable, timely, and personalized opportunity to increase your
     # sales and performance.
     class Client < ::Peddler::Client
-      path '/Recommendations/2013-04-01'
+      version '2013-04-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.